pkg/telemetry/prometheus/node_nonlinux.go

```diff
@@ -34,6 +35,16 @@ func getCPUStats() (cpuLoad float32, numCPUs uint32, err error) {
 	return
 }
 
+func getMemoryStats() (memoryLoad float32, err error) {
```

Comment:
If the code for Linux and non Linux is the same, you can move it into node.go

Reply:
Oops, sorry, missed this comment. Yeah, it is the same. Will make another small PR to address that.